distribution/ddf/pom.xml

```diff
@@ -260,6 +260,11 @@
             <type>xml</type>
         </dependency>
         <dependency>
+            <groupId>ddf.platform.osgi</groupId>
+            <artifactId>platform-osgi-configadmin</artifactId>
```

Comment:
❓ I guess it depends on where we think these modules may be going in the future, but I'm wondering if we should be using the word "felix" instead of "osgi" since our code is so specific to Felix's implementation?

Reply:
The fact that we're embedding felix is now an implementation detail. The point I'm trying to convey is that we now own the impl for osgi's config admin.